### ISSUE TITLE
Lane line sensor source code

### ISSUE THREAD

**Issue by ertauren** [lgsvl/simulator#1856]
Hi,
I am interested in [Lane line sensor](https://www.svlsimulator.com/docs/simulation-content/lane-line-sensor/)'s source code.
But I couldn't find it on [Repositories](https://github.com/orgs/lgsvl/repositories)
Is there any plan to share it?

**Comment by EricBoiseLGSVL** [comment]
No, sorry we don't have that open sourced at the moment.  I'll see if we have plans to.

**Comment by EricBoiseLGSVL** [comment]
@lemketron or @lokesku
